main.tf

```
@@ -3,7 +3,7 @@ terraform {
 }
 
 module "lambda" {
-  source = "git::https://github.com/plus3it/terraform-aws-lambda.git?ref=v1.3.0"
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-lambda.git"
```

Keep the ref, just updated for the latest release of that project:

```suggestion
  source = "git::https://github.com/terraform-aws-modules/terraform-aws-lambda.git?ref=v5.0.0"
```